src/Option_Command.php

```diff
@@ -273,8 +293,16 @@ public function list_( $args, $assoc_args ) {
 		}
 		$where .= $autoload_query . $transients_query;
 
-		$results = $wpdb->get_results( "SELECT `option_name`,`option_value`,`autoload`" . $size_query
-					. " FROM `$wpdb->options` {$where}" );
+		// Ser orderby field.
+		$orderby_field = 'option_name';
+		if ( 'option_value' === \WP_CLI\Utils\get_flag_value( $assoc_args, 'orderby' ) ) {
+			$orderby_field = 'option_value';
+		} else if ( 'option_id' === \WP_CLI\Utils\get_flag_value( $assoc_args, 'orderby' ) ) {
+			$orderby_field = 'option_id';
+		}
+
+		$results = $wpdb->get_results( "SELECT `option_id`,`option_name`,`option_value`,`autoload`" . $size_query
+					. " FROM `$wpdb->options` {$where} ORDER BY `$wpdb->options`.`$orderby_field` $sort" );
```

Comment:
The `order_by` should be optional, so that when you don't specify it, you get the results in the order that they are stored in the DB. This is better for performance and keeps the current existing behavior intact.